modules/seo/helpers-seo.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/**
+ * Used to check whether advanced seo features are enabled.
+ *
+ * @return bool True if advanced seo features are enabled, false otherwise.
+ */
+function is_enabled_advanced_seo() {
+	// stub
+	return true;
+}
+
+/**
```

Comment:
On wpcom side these functions were namespaced, but I removed that in order to preserve backwards compatibility with PHP 5.2 (is it still a requirement?). I was considering refactoring all SEO helpers functions as static methods of a single class. I also don't mind prefixing them if that is the preferred way to do it. 
(same holds for comments bellow)